homeassistant/components/nzbget/sensor.py

```diff
@@ -105,17 +105,24 @@ def __init__(
         description: SensorEntityDescription,
     ) -> None:
         """Initialize a new NZBGet sensor."""
-        self.entity_description = description
-        self._attr_unique_id = f"{entry_id}_{description.key}"
-
         super().__init__(
             coordinator=coordinator,
             entry_id=entry_id,
             name=f"{entry_name} {description.name}",
         )
 
+        self.entity_description = description
+        self._attr_unique_id = f"{entry_id}_{description.key}"
+        self._attr_native_value = self._native_value
+
+    @callback
+    def _handle_coordinator_update(self) -> None:
+        """Handle updated data from the coordinator."""
+        self._attr_native_value = self._native_value
+        super()._handle_coordinator_update()
+
     @property
-    def native_value(self):
+    def _native_value(self):
```

Comment:
I'd name the new attribute `self._native_value` and keep the property `native_value`. Then we don't need to implement `_handle_coordinator_update`.